build.gradle

```diff
@@ -1,18 +1,94 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.5.1.RELEASE")
+        classpath 'org.hidetake:gradle-ssh-plugin:1.1.3'
+    }
+}
+
+version '1.0.3-SNAPSHOT'
+
 apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven'
+apply plugin: 'eclipse'
```

Comment:
Do we need this 'eclipse' plugin?
Here is the docs: https://docs.gradle.org/3.3/userguide/eclipse_plugin.html
If not, then better to remove it.